### ISSUE TITLE
The truth value of an array with more than one element is ambiguous. Use a.any() or a.all()

### ISSUE THREAD

**Issue by Erik-02** [tblume1992/ThymeBoost#27]
When using thymeboost.autofit method, I am using a column in a data frame as follows:
output = boosted_model.autofit(train[column], seasonal_period=[3,4,6,12]).
Then I get this error: The truth value of an array with more than one element is ambiguous. Use a.any() or a.all()

What is weird is that when I am using this method in Jupyter notebook, on python=3.7.16, thymeboost=0.1.15, it all works fine.
Although not ideal since other libraries that I am using need python=3.8 or higher.
However, as soon as I try to use it in VScode, python=3.10.11, thymeboost=0.1.15, it does not work anymore and I get this error.

The values are just regular integers. I have tried downgrading to previous versions of thymeboost but it still does not work.
What did work: I started a new venv, just like before, although this time with python=3.8, thymeboost=0.1.15, and it seems to work perfectly again. Maybe an issue with python 3.10.11?

**Comment by tblume1992** [comment]
Yeah could be a 3.10 issue, can you send a full trace back? Are the env using different versions of pandas as well?
